plugins/doc_fragments/VmwareRestModule.py

```diff
@@ -15,22 +15,24 @@ class ModuleDocFragment(object):
       description:
       - Indicates whether get_id() can return multiple IDs for a given name.
       - Typically, this should be false when updating or deleting; otherwise, all named objects could be affected.
-      required: true
+      default: false
       version_added: "2.10"
+      version_added_collection: 'ansible.builtin'
       type: bool
     log_level:
       description:
       - If ANSIBLE_DEBUG is set, this will be forced to 'debug', but can be user-defined otherwise.
-      required: True
       choices: ['debug', 'info', 'normal']
       version_added: "2.10"
+      version_added_collection: 'ansible.builtin'
```

Comment:
I've not seen `version_added_collection` before, what does this mean?

Reply:
This feature is added in Ansible 2.10. I used it to indicate that this was added in Ansible 2.10. for maintaining historical data.

Reply:
We usually replace `version_added: 2.10` with `version_added: <version of this collection>` since "added in 2.10" should mean "added in this collection" (even if the stuff was migrated from ansible/ansible). This distinction is important since this collection can be used without ansible 2.10 (by using ansible-base 2.10, or even by using ansible 2.9 if no technical reason prevents this).

Reply:
(Anyway, this is a style question, and since I'm not involved in this collection I don't care how things are done here ;) )